scripts/tools/CheckDegradations.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA;
+ *
+ */
+
+namespace oat\taoSystemStatus\scripts\tools;
+
+use oat\generis\persistence\PersistenceManager;
+use oat\oatbox\extension\script\ScriptAction;
+use oat\taoSystemStatus\model\Report\ReportComparator;
+use oat\taoSystemStatus\model\SystemStatus\SystemStatusService;
+use common_report_Report as Report;
+
+/**
+ * Class CheckDegradations
+ *
+ * ```bash
+ * $ sudo -u www-data php index.php '\oat\taoSystemStatus\scripts\tools\CheckDegradations' -p default_kv
+ * ```
+ *
+ * @package oat\taoSystemStatus\scripts\tools
+ * @author Aleh Hutnikau, <hutnikau@1pt.com>
+ */
+class CheckDegradations extends ScriptAction
+{
+    private const KV_PERSISTENCE_KEY = self::class.'::systems_status_report';
+
+    /**
+     * @return array[]
+     */
+    public function provideOptions()
+    {
+        return [
+            'persistence' => [
+                'prefix' => 'p',
+                'longPrefix' => 'persistence',
+                'required' => true,
+                'description' => 'The KeyValue persistence where you want store results of previous check',
```

Comment:
As a newbie with this code, it would greatly help my understanding if I could see the available values, or an example.